features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/data/api.jsx

```diff
@@ -367,10 +367,23 @@ export default class API extends Resource {
         }
     }
 
+    removeRatingOfUser(apiId, callback = null) {
+        const promiseDelete = this.client.then((client) => {
+            return client.apis.Ratings.delete_apis__apiId__user_rating({ apiId }, this._requestMetaData());
+        }).catch((error) => {
+            console.error(error);
+        });
+        if (callback) {
+            return promiseDelete.then(callback);
+        } else {
```

Comment:
This callback is not required here, We have used this pattern before, but it is a redundant work because we are returning the promise object anyway

Reply:
Fixed